src/wazuh_modules/wm_database.c

```diff
@@ -890,11 +883,6 @@ static void * wm_inotify_start(__attribute__((unused)) void * args) {
                     mterror(WM_DATABASE_LOGTAG, "Inotify event too large (%u)", event->len);
                     break;
                 }
-
-                if (event->name[0] == '.') {
-                    mtdebug2(WM_DATABASE_LOGTAG, "Discarding hidden file.");
```

Comment:
The implications of the hidden files should be discussed with the team

Reply:
We should allow only hidden folders and not hidden files to avoid unnecessary **inotify** events.

Reply:
As we agreed after analyzing this, the piece of code was restored with the inclusion of a check to only discard hidden files events but handle hidden directory events.